sdk/appcenter-distribute/src/main/java/com/microsoft/appcenter/distribute/DistributeUtils.java

```diff
@@ -47,15 +55,62 @@ class DistributeUtils {
      */
     static final String TESTER_APP_PACKAGE_NAME = "com.microsoft.hockeyapp.testerapp";
 
+    /**
+     * Notification channel identifier.
+     */
+    static final String NOTIFICATION_CHANNEL_ID = "appcenter.distribute";
+
     /**
      * Get the notification identifier for downloads.
      *
      * @return notification identifier for downloads.
      */
+    @VisibleForTesting
     static int getNotificationId() {
         return Distribute.class.getName().hashCode();
     }
 
+    static void postDownloadNotification(@NonNull Context context, String message, Intent intent) {
+        NotificationManager notificationManager = (NotificationManager) context.getSystemService(Context.NOTIFICATION_SERVICE);
+        Notification.Builder builder;
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+
+            /* Create or update notification channel (mandatory on Android 8 target). */
```

Comment:
on Android 8 and higher?

Reply:
Sure, that what is comment and code about